ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -534,8 +534,30 @@ tests:
       ZONES_COUNT: "3"
     test:
     - chain: openshift-e2e-test-qe
-    workflow: cucushift-installer-rehearse-aws-ipi-edge-zone-byo-subnet-cco-manual-security-token-service
-- as: aws-ipi-localzone-byo-subnet-role-sts-f28-destructive
+    workflow: cucushift-installer-rehearse-aws-ipi-edge-zone-byo-subnet-cco-manual-security-token-service-byo-kms
+- as: aws-ipi-localzone-byo-subnet-sts-kms-reenc-f28-destructive
+  cron: 52 0 10 * *
+  steps:
+    cluster_profile: aws-1-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      EDGE_ZONES_LIST: us-east-1-mia-1a
+    test:
+    - chain: openshift-e2e-test-qe-destructive
+    workflow: cucushift-installer-rehearse-aws-ipi-edge-zone-byo-subnet-cco-manual-security-token-service-byo-kms
+- as: aws-ipi-localzone-byo-subnet-sts-kms-reenc-f14
```

Comment:
Compared with `aws-ipi-localzone-byo-subnet-role-sts-kms-mini-perm-f14`,  the only difference on this job is specifying `EDGE_ZONES_LIST` to a particular zone, right ?

Reply:
Yes. The issue will occur in this zone if no `kms:ReEncrypt*` permission in Mint and STS mode.

Reply:
Can we leverage the configuration into the existing `aws-ipi-localzone-byo-subnet-role-sts-kms-mini-perm` ci jobs ? E.g: non-destructive ci job specifies the particular zone, and destructive ci job does not specify it.

Reply:
`aws-ipi-localzone-byo-subnet-role-sts-kms-mini-perm` is updated from byo-subnet-role-sts w/o kms configuration, a random selected LZ is used, so all LZs will be covered in this job, and both non-destructive and destructive jobs are important I think. `kms-reenc` jobs focus on  [OCPBUGS-56159](https://issues.redhat.com/browse/OCPBUGS-56159) issue only, but we can decrease the frequency and/or remove destructive job.